src/types/hashed.ts

```diff
@@ -26,11 +18,29 @@ export const validate: TypeUtils.Validate<Types['Write'], DbWriteType> =
 		if (typeof value !== 'string') {
 			throw new Error('is not a string');
 		}
-		const salt = await bcrypt.genSalt();
-		return bcrypt.hash(value, salt);
+		const salt = crypto.randomBytes(16).toString('hex');
+		return new Promise((resolve, reject) => {
```

Comment:
How about trying `https://www.npmjs.com/package/@node-rs/bcrypt` so that we don't need to do this breaking change?